src/electron/process_manager.ts

```diff
@@ -240,7 +242,7 @@ function startLocalShadowsocksProxy(
     // Note that if you run with -v then ss-local may output a lot of data to stderr which
     // will cause the binary to fail:
     //   https://nodejs.org/dist/latest-v10.x/docs/api/child_process.html#child_process_maxbuffer_and_unicode
-    ssLocal = spawn(pathToEmbeddedBinary('ss-local'), ssLocalArgs);
+    ssLocal = spawn(pathToEmbeddedBinary('shadowsocks-libev', 'ss-local'), ssLocalArgs);
```

Comment:
Did ss-local get renamed? Isn't it clearer to keep the original name?

Reply:
The layout on disk changed: previously, everything was jammed into the top of the install directory. Now, the directory tree is mirrored there - so the paths have to be more exact but do not change between development and install time.